### ISSUE TITLE
Fix accessibility issues raised by accessibility evaluation

### ISSUE THREAD

**Issue by cbunicef** [unicef/iogt#1622]
- In all pages there is no "skip to main content" link at the to of the page
- Text on the homepage article "parent section" does not have sufficient color contrast ratio
- Language selector is not keyboard accessible
- Order of columns in navigation is different than visual order
- The form is not accessible. The screen reader is not able to announce its title and the question goes directly to the combo box.

**Comment by cbunicef** [comment]
Here is an example of "skip to main content": https://webaim.org/, press tab on keyboard to show the first navigation element.

We could enable this using JS, which ideally would mean that it doesn't appear on feature phones which mostly won't have screen reading capabilities (need to confirm?)
